modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -263,6 +263,10 @@
 	z-index: 100 !important;
 }
 
+.cke_panel.cke_menu_panel {
+	z-index: 10001 !important;
+}
+
```

Comment:
All these `!important`s make me nervous. It is a bad pattern to start having them everywhere, as future overrides are now much harder. It is a joker of sorts to be used when we're out of all other options. The only saving grace here is it is just `z-index`, that already has messed up priorities handling.